src/main/java/core/basesyntax/service/StorageServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
```

Comment:
let's use StorageService only for adding and getting information from Storage, other logic should be moved to handlers of operations